src/components/Elements/Map/MapScreen.tsx

```diff
@@ -324,6 +324,7 @@ const MapScreen = (): JSX.Element => {
         // bottomSheetRef.current?.snapToIndex(1)
         _removeAllGeoJson(mapRef)
         _addGeoJson()
+        _injectMarker(mapRef, mapCenter, colors)
     }, [currentFloor, allRooms, colors, availableRooms, allRooms, loadingState])
```

Comment:
This alos injects the marker on initial load. Try adding something like  `
if (clickedElement != null) {}` to fix it